components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -505,6 +505,26 @@ - (void)testBottomDrawerScrollingEnabled {
   XCTAssertEqual(self.fakeScrollView.scrollEnabled, NO);
 }
 
+- (void)testSetTrackingScrollView {
+  // Given
+  MDCBottomDrawerPresentationController *drawerPresentationController = nil;
+  if ([self.drawerViewController.presentationController
+          isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
+    drawerPresentationController =
+        (MDCBottomDrawerPresentationController *)self.drawerViewController.presentationController;
+  } else {
+    drawerPresentationController = nil;
+  }
+
+  // When
+  self.drawerViewController.scrimColor = UIColor.blueColor;
```

Comment:
Why is it important to make the scrim color blue?  Maybe adding some comments here will help clarify the test?

Reply:
If a client sets the `scrimColor`, `isTopHandleHidden` or `topHandleColor` before setting the tracking scroll view it used to not set the tracking scroll view. I just picked `scrimColor` at random.

Reply:
Ok. Please add a comment to explain that's the reason. I thought it was debugging code left-over in the test.

Reply:
Done